### PR TITLE
Project ARES: Reception Remodel

### PR DESCRIPTION
**Pull request by realforest2001**
# About the pull request
Remodels the AI Core Reception Desk to be behind a window. Also adds item chutes to depost visitor weapons into a vault.

# Explain why it's good for the game

Being behind the window makes things a little more thematic. Further, the downstairs remodel gives a bit more space.

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl:
maptweak: Remodelled the AI Core Reception and WJ spawn area.
maptweak: Switches AI Core cameras over to manual naming.
imageadd: Added sprites for AI Core windows on black Almayer frames.
/:cl:

### CODE REVIEW

File: maps/map_files/USS_Almayer/USS_Almayer.dmm
Comment (Nanu308) [inline maps/map_files/USS_Almayer/USS_Almayer.dmm:1]: Mapping is fine, a bit unsure about the right stairwell but we'll see it in action if it becomes an issue

**Review by Nanu308 — APPROVED** [review]

**Review by sleepynecrons — APPROVED** [review]
sprites fine

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Review by Drulikar — APPROVED** [review]

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Pull request merged**